saturn/environment.yml

```diff
@@ -31,7 +31,9 @@ dependencies:
 - protobuf
 - xlrd
 - ipykernel
-- dask
+- dask=2.8.1
```

Comment:
you have to make the changes here actually

https://github.com/saturncloud/images/blob/master/templates/saturn/environment.yml